smartsim/_core/config/config.py

```diff
@@ -204,6 +204,17 @@ def test_account(self) -> t.Optional[str]:  # pragma: no cover
         # no account by default
         return os.environ.get("SMARTSIM_TEST_ACCOUNT", None)
 
+    @property
+    def telemetry_frequency(self) -> int:
+        return int(os.environ.get("SMARTSIM_TELEMETRY_FREQUENCY", 5))
+
+    @property
+    def telemetry_enabled(self) -> bool:
+        return bool(os.environ.get("SMARTSIM_FLAG_TELEMETRY", 1))
```

Comment:
Per discussion: Turn this off by default

Reply:
Default modified.